Userland/Applications/AnalogClock/AnalogClock.h

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2021, Erlend Høier <Erlend@ReasonablePanic.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <LibGUI/Painter.h>
+#include <LibGUI/Widget.h>
+
+class AnalogClock : public GUI::Widget {
+    C_OBJECT(AnalogClock)
+public:
+    ~AnalogClock() override = default;
+
+private:
+    AnalogClock()
+        : m_small_graduation_square(Gfx::IntRect({}, { 3, 3 }))
+        , m_big_graduation_square(Gfx::IntRect({}, { 5, 5 }))
+    {
+        start_timer(25);
+    };
+
+    int m_test { 0 };
+    int m_test_2 { 0 };
```

Comment:
I guess you forgot to remove these 👀

Reply:
Thank you :) I also noticed that I'm updating every 25 ms instead of 1000ms (also a remnant of testing) .

Reply:
You removed the variables but you're still updating every 25ms 🙈

Reply:
Yeah thanks :)  I was focused on removing the "merge commit" from the PR